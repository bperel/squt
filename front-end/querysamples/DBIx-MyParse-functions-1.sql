SELECT CASE 3 WHEN 1 THEN 'one' WHEN 2 THEN 'two' ELSE 'more' END
