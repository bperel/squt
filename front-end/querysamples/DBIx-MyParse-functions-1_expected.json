{
  "Warning": {
    "No alias": {
      "case_switch": ["function", "SELECT"]
    }
  },
  "Functions": {
    "0": {
      "to": "OUTPUT",
      "Constants": {
        "1": "1",
        "one": "one",
        "3": "3",
        "more": "more",
        "two": "two",
        "2": "2"
      },
      "name": "case_switch",
      "alias": "0"
    }
  }
}
